week-1/L-functions-nested/exercise2.js

```diff
@@ -0,0 +1,20 @@
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+//
+function stringToUpperCase(myName){
```

Comment:
Think about better naming for your functions. What does this function do ultimately? When someone looks at the code and reads "stringToUpperCase", is it obvious it will do that?

Reply:
Yes, I will.